README.md

```diff
@@ -224,6 +224,7 @@ ros2 launch gazebo_ros2_control_demos cart_example_position.launch.py
 ros2 launch gazebo_ros2_control_demos cart_example_velocity.launch.py
 ros2 launch gazebo_ros2_control_demos cart_example_effort.launch.py
 ros2 launch gazebo_ros2_control_demos diff_drive_example.launch.py
+ros2 launch gazebo_ros2_control_demos tricycle_drive_example.launch.py
```

Comment:
```suggestion
ros2 launch gazebo_ros2_control_demos tricycle_drive.launch.py
```

Reply:
Ah sorry about that, thanks

Reply:
Ah, then the diff drive one is also wrong